Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -13,60 +9,67 @@ public class PayPalClient {
 
     let config: CoreConfig
 
-    // swiftlint:disable identifier_name
-    let CheckoutFlow: CheckoutProtocol.Type
-    // swiftlint:enable identifier_name
+    let nativeCheckout: CheckoutProtocol
 
     /// Initialize a PayPalClient to process PayPal transaction
     /// - Parameters:
     ///   - config: The CoreConfig object
     ///   - returnURL: The return URL provided to the PayPal Native UI experience. Used as part of the authentication process to identify your application. This value should match the one set in the `Return URLs` section of your application's dashboard on your [PayPal developer account](https://developer.paypal.com)
     public init(config: CoreConfig) {
         self.config = config
-        self.CheckoutFlow = Checkout.self
+        let nxoConfig = CheckoutConfig(
+            clientID: config.clientID, createOrder: nil, onApprove: nil,
+                    onShippingChange: nil, onCancel: nil, onError: nil, environment: config.environment.toNativeCheckoutSDKEnvironment())
```

Comment:
Newlines seem off here, I think every parameter should be on a newline since the first one is.

Reply:
+1